locale/tr/wivrn-dashboard.po

```diff
@@ -653,22 +663,36 @@ msgstr ""
 msgid "I cannot see my computer in the WiVRn app"
 msgstr "WiVRn uygulamasında bilgisayarımı göremiyorum"
 
-#: dashboard/qml/TroubleshootPage.qml:25 dashboard/qml/WizardPage.qml:393
+#: dashboard/qml/TroubleshootPage.qml:25
+#, kde-format
+msgid "If you have a firewall, make sure that port 5353/UDP is open."
+msgstr "Güvenlik duvarı var ise, 5353/UDP port'u açık olduğuna emin olun."
+
+#: dashboard/qml/TroubleshootPage.qml:28
+#, kde-format
+msgid "Connecting remains stuck on \"Connection to ...\""
+msgstr "Bağlantı \"Bağlanılıyor...\"'da takılı kaldı"
+
+#: dashboard/qml/TroubleshootPage.qml:29 dashboard/qml/TroubleshootPage.qml:33
 #, kde-format
 msgid ""
-"If the server is not visible or the connection fails, check that port 5353 "
-"(UDP) and 9757 (TCP and UDP) are open in your firewall."
+"If you have a firewall, make sure that port 9757 is open both for TCP and "
+"UDP."
 msgstr ""
-"Sunucu görünür değilse veya bağlantı başarısız oluyorsa, güvenlik "
-"duvarınızda 5353 (UDP) ve 9757 (TCP ve UDP) bağlantı noktalarının açık "
-"olduğundan emin olun."
+"Güvenlik duvarı var ise, hem TCP hemde UDP için 9757 port'u açık olduğuna "
+"emin olun."
 
-#: dashboard/qml/TroubleshootPage.qml:28
+#: dashboard/qml/TroubleshootPage.qml:32
+#, kde-format
+msgid "Connection fails with \"no route to host\""
+msgstr "Bağlantı başarısız oldu, \"ana bilgisayara bağlantı yok\""
```

Comment:
The "no route to host" on headset is not translated (it's directly the system error)